### PR TITLE
SC-2049:Language Upgrade - Java 11

### PR DESCRIPTION
**Pull request by aishwa8141**

### CODE REVIEW

**Review by PradyumnaNagendra — APPROVED** [review]

File: all-actors/pom.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.7.5.201505241946</version>
+                <version>0.8.5</version>
```
Comment (maheshkumargangula) [inline all-actors/pom.xml:43]: Please create `property` and use that for version value.

File: es-utils/pom.xml [hunk 2]
```diff
@@ -63,7 +63,7 @@
 				<plugin>
 					<groupId>org.jacoco</groupId>
 					<artifactId>jacoco-maven-plugin</artifactId>
-					<version>0.7.5.201505241946</version>
+					<version>0.8.5</version>
```
Comment (maheshkumargangula) [inline es-utils/pom.xml:66]: Please create `property` and use that for version value.

**Review by maheshkumargangula — CHANGES_REQUESTED** [review]

File: all-actors/pom.xml (hunk 1 above)
Comment (aishwa8141) [reply all-actors/pom.xml]: done

File: es-utils/pom.xml (hunk 2 above)
Comment (aishwa8141) [reply es-utils/pom.xml]: done

**Review by maheshkumargangula — APPROVED** [review]

**Pull request merged**